lib/sign-in-component.js

```diff
@@ -31,6 +30,13 @@ class SignInComponent {
     return etch.update(this)
   }
 
+  readAfterUpdate () {
```

Comment:
Because the view refreshes, we have to rewire the `onDidChange` event otherwise the sign in button will be stuck in a disabled state, even when a token value is input.